test/helpers/dummyData.js

```diff
@@ -0,0 +1,40 @@
+export const user1 = {
+  username: 'flippin',
+  email: 'flipping2234@gmail.com',
+  name: 'Flipping James',
+  password: '1234567'
+};
+
+export const user2 = {
+  username: 'flippingg',
+  email: 'flippingg2234@gmail.com',
+  name: 'Flippingg James',
+  password: '1234567g'
+};
+
+export const article1 = {
+  title: 'This is the title a test article',
+  description: 'This is the description of the test article',
+  body:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
+};
+
+export const validUser = {
+  username: 'JamesBond',
+  email: 'jamesbondxxc@gmail.com',
+  name: 'James Bond',
+  password: '1234567'
+};
+
+export const helperUser = {
+  username: 'AyinlaOlajide',
+  email: 'helperuser@gmail.com',
+  name: 'Ayinla Olajide',
+  password: '1234567'
+};
+
+export const profileDetails = {
```

Comment:
@olajide1234 let's have different files for different dummy data. One for `user` related data, another for `articles` etc.
Once that is done, this is ready for merge

Reply:
Implemented as requested